sbin/common/config_init.sh

```diff
@@ -366,6 +367,35 @@ function configDefaults() {
   # The OS kernel name, e.g. 'darwin' for Mac OS X
   BUILD_CONFIG[OS_KERNEL_NAME]=$(uname | awk '{print tolower($0)}')
 
+  # Determine OS full system version
+  local unameSys=$(uname -s)
+  local unameOSSysVer=$(uname -sr)
+  if [ "${unameSys}" == "Linux" ]; then
+    # Linux distribs add more useful distrib /etc/*-release file content
+    set +e
+    cat /etc/*-release > /dev/null
+    local cmdRc="$?"
+    if [ "${cmdRc}" == "0" ]; then
+      local linuxName=""
+      cat /etc/*-release | grep PRETTY_NAME > /dev/null
+      cmdRc="$?"
+      if [ "${cmdRc}" == "0" ]; then
+        linuxName=$(cat /etc/*-release | grep PRETTY_NAME | cut -d'=' -f2 | tr -d '"')
+      else
+        linuxName=$(cat /etc/*-release | tr -d '"')
+      fi
+      unameOSSysVer="${unameOSSysVer} : ${linuxName}"
+    fi
+    set -e
+  elif [ "${unameSys}" == "AIX" ]; then
+    # AIX provides full version info using oslevel
+    aixVer=$(oslevel)
```

Comment:
This gives a more useful definition of the OS level
```suggestion
    aixVer=$(oslevel -r)
```

Reply:
yep